lib/BirthdayCalendarGenerator.php

```diff
@@ -136,6 +136,11 @@ function getResult() {
                 continue;
             }
 
+            // Skip if the card has no FN property.
+            if (!isset($object->FN)) {
```

Comment:
I guess we should re-order this "skip if" statements from the cheapest to the expensive ones